recipes/pandas/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version="0.18.1" %}
+
+package:
+    name: pandas
+    version: {{ version }}
+
+source:
+    fn: pandas-{{ version }}.tar.gz
+    url: https://github.com/pydata/pandas/archive/v{{ version }}.tar.gz
+    sha256: 1f3136f69a58d490ba9f9874e67601042a11138270515dedcf6979c003bba79b
+
+build:
+    number: 0
+    script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - cython
+    - numpy             x.x
+    - setuptools
+    - pytz
+    - python-dateutil
+
+  run:
+    - python
+    - numpy             x.x
+    - python-dateutil
+    - pytz
+
+test:
+  imports:
+    - pandas
+
+about:
+  home: http://pandas.pydata.org
+  license: BSD
```

Comment:
Should be [BSD 3-Clause]( http://pandas.pydata.org/pandas-docs/stable/overview.html#license ).